apps/alert_processor/lib/model/query.ex

```diff
@@ -234,6 +234,18 @@ defmodule AlertProcessor.Model.Query do
           UNION ALL
           SELECT 'Paused for 12 months or more',plmore.total_paused_accounts FROM paused_for_twelve_months_or_more plmore;
         """
+      },
+      %__MODULE__{
+        label: "Count of Active Users Having Subscriptions That Are Parents",
```

Comment:
thought (non-blocking): I wonder if there's some wording that makes this label more generally understandable / less specific to the implementation details. Maybe something along the lines of, "parallel route subscriptions"?

Reply:
Or even, "parallel bus route subscriptions" since this is only possible with bus routes?